python/cudf/cudf/core/dataframe.py

```diff
@@ -3248,127 +3215,6 @@ def label_encoding(
 
         return outdf
 
-    @annotate("ARGSORT", color="yellow", domain="cudf_python")
```

Comment:
I assume we should be safe to remove these annotations right? Now that `nvtx` auto-annotates? 

cc: @shwina for visibility

Reply:
NVTX does not auto-annotate. There's an open PR to opt-in to auto annotation (https://github.com/NVIDIA/NVTX/pull/32).

We likely want to keep these annotations. Tangentially, we should probably remove annotations from non-public methods and be consistent about annotating all public  methods.